config.json

```diff
@@ -1031,6 +1031,11 @@
         "Lists"
       ]
     },
+      {
```

Comment:
I added the uuid, core and unlocked_by values as requested.  I also added additional topics.   By the way,  I came across this issue for the exercise configlet https://github.com/exercism/configlet/issues/30.   According to this issue, the uuid values are supposed to be globally unique.   I generated a version 1 uuid from uuidgenerator.net and applied this value instead of copying the value from the Java track